frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx

```diff
@@ -94,8 +94,8 @@ export const StorageClassFormProvisoners: ExtensionSCProvisionerProp = Object.fr
         },
         thickProvision: {
           name: 'Enable Thick Provisioning',
+          hintText: 'Volumes will allocate the requested capacity upon volume creation',
```

Comment:
externalize

Reply:
This is an `Object.freeze` object. How should I externalize it ?? This object is defined in `ocs-storage-class-params.tsx`

Reply:
https://github.com/openshift/console/blob/ef3572aba2ac2feccb1bd64d9fa90086127de7ac/frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx#L97